setup.py

```diff
@@ -121,10 +121,12 @@
         "conllu==2.2.2",
         "transformers>=2.4.0",
         "jsonpickle",
+        "semantic_version",
```

Comment:
Wait, I just realized that `semantic_version` shouldn't be in the regular dependencies, and also that it's missing from `dev-requirements.txt` :grimacing: